blockmanager.go

```diff
@@ -1055,8 +1055,8 @@ func (b *blockManager) writeCFHeadersMsg(msg *wire.MsgCFHeaders,
 	headerBatch[numHeaders-1].HeaderHash = lastHash
 	headerBatch[numHeaders-1].Height = lastHeight
 
-	log.Debugf("Writing filter headers up to height=%v, hash=%v",
-		lastHeight, lastHash)
+	log.Debugf("Writing filter headers up to height=%v, hash=%v, "+
+		"new_tip=%v", lastHeight, lastHash, lastHeader)
```

Comment:
As this is a header (the new formatting argument), perhaps we should use spew to display it properly?

Reply:
It is actually just a hash.